pkg/controllers/binding/common.go

```diff
@@ -48,26 +48,21 @@ func ensureWork(
 ) error {
 	bindingSpec := getBindingSpec(binding, scope)
 	targetClusters := mergeTargetClusters(bindingSpec.Clusters, bindingSpec.RequiredBy)
-
-	var jobCompletions []workv1alpha2.TargetCluster
 	var err error
-	if workload.GetKind() == util.JobKind {
-		jobCompletions, err = divideReplicasByJobCompletions(workload, targetClusters)
-		if err != nil {
-			return err
-		}
-	}
-
 	var errs []error
 	for i := range targetClusters {
 		targetCluster := targetClusters[i]
 		clonedWorkload := workload.DeepCopy()
 
 		workNamespace := names.GenerateExecutionSpaceName(targetCluster.Name)
 
-		// If and only if the resource template has replicas, and the replica scheduling policy is divided,
-		// we need to revise replicas.
-		if needReviseReplicas(bindingSpec.Replicas, bindingSpec.Placement) {
+		// If and only if the resource template has replicas, we need to revise replicas.
+		// The replicas value in "Work" should be consistent with the scheduling result bindingSpec.Clusters,
+		// or bindingSpec.RequiredBy controlled by the dependencies-distributor, rather than bindingSpec.Replicas.
+		// This remains true even when the replica scheduling policy is duplicated.
+		// In certain scenarios, such as scheduling suspension or insufficient federation quota,
+		// bindingSpec.Replicas should not be directly synchronized to the member clusters.
+		if needReviseReplicas(bindingSpec.Replicas) {
```

Comment:
I might be missing something, but should we perhaps reorder `needReviseReplicas` and  `needReviseJobCompletions`? At the moment, since jobs have replicas, `needReviseReplicas` will return true and skip the `needReviseJobCompletions` segment altogether if ReviseReplica goes through without errors.